cluster/pre-existing/util.sh

```diff
@@ -44,19 +44,23 @@ function create-certs {
   rm /tmp/kubeconfig
 
   execute-cmd-on-pre-existing-master-with-retries "sudo cat /etc/kubernetes/admin.conf" > /tmp/kubeconfig
-  CA_CERT_BASE64=$(cat /tmp/kubeconfig | grep certificate-authority | awk '{print $2}' | head -n 1)
-  KUBELET_CERT_BASE64=$(cat /tmp/kubeconfig | grep client-certificate-data | awk '{print $2}' | head -n 1)
-  KUBELET_KEY_BASE64=$(cat /tmp/kubeconfig | grep client-key-data | awk '{print $2}' | head -n 1)
+  CA_CERT_BASE64=$(grep certificate-authority /tmp/kubeconfig | awk '{print $2}' | head -n 1)
+  export CA_CERT_BASE64
+  KUBELET_CERT_BASE64=$(grep client-certificate-data /tmp/kubeconfig | awk '{print $2}' | head -n 1)
+  KUBELET_KEY_BASE64=$(grep client-key-data /tmp/kubeconfig | awk '{print $2}' | head -n 1)
 
   # Local kubeconfig.kubemark vars
   KUBECFG_CERT_BASE64="${KUBELET_CERT_BASE64}"
+  export KUBECFG_CERT_BASE64
   KUBECFG_KEY_BASE64="${KUBELET_KEY_BASE64}"
+  export KUBECFG_KEY_BASE64
 
   # The pre-existing Kubernetes master already has these setup
   # Set these vars but don't use them
   CA_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  export CA_KEY_BASE64
   MASTER_CERT_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  export MASTER_CERT_BASE64
   MASTER_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
-  KUBEAPISERVER_CERT_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
-  KUBEAPISERVER_KEY_BASE64=$(dd if=/dev/urandom bs=128 count=1 2>/dev/null | base64 | tr -d "=+/" | dd bs=32 count=1 2>/dev/null)
+  export MASTER_KEY_BASE64
```

Comment:
confirming that the rest of these are in fact used by something else?

Reply:
I've checked.
I think the following variables are not used by anywhere.
- KUBECFG_CERT_BASE64
- KUBECFG_KEY_BASE64
- CA_KEY_BASE64
- MASTER_CERT_BASE64
- MASTER_KEY_BASE64

KUBECFG_* and MASTER_* were used in start-kubemark.sh, but https://github.com/kubernetes/kubernetes/pull/78246 removed them from start-kubemark,sh.
CA_KEY_BASE64 is defined and used by cluster/gce/util.sh.

Therefore, I'm thinking about removing these variables. Can I get your opinion?

Reply:
if we're confident that they are not used we should remove them :-)

it looks pretty likely that we can remove all of these including CA_KEY_BASE64 (from this script anyhow)

Reply:
Thanks. I updated the commit. In the commit, I removed these variables.

On the other hand, CA_CERT_BASE64, KUBELET_CERT_BASE64 and KUBELET_KEY_BASE64 remain because it may be used by test/kubemark/iks/util.sh.

Reply:
IKS is for "ibm kubernetes service", does it really use this file?

cc @Huang-Wei

Reply:
For some historical reason, the IKS kubemark scripts follows the GCE kubemark scripts to implemented IKS's specific functions. Now we have #78246, it seems the IKS scripts has fallen behind and needs some cleanup.

I need to take a thorough look into #78246 and then clean up the IKS scripts a bit. After that, we should be able to remove CA_KEY_BASE64.

At this moment, can we keep the IKS scripts as is? (But it's ok to remove CA_KEY_BASE64 from cluster/gce/util.sh, it appropriate)

Reply:
@Huang-Wei is the IKS script used with this one? This PR is not trying to change the IKS script but we want to drop these variables if they are not being used.

Reply:
@BenTheElder  I don't think IKS script used `cluster/pre-existing/util.sh`.

Reply:
Thanks for many comments.
I rechecked it from a different perspective.
CA_CERT_BASE64 is defined by create-certs() in pre-existing/util.sh.
create-certs() is used by only cluster/gce/util.sh. But it defines the function by itself.

When pre-existing/utils.sh was created, start-kubemark.sh used create-certs(). https://github.com/kubernetes/kubernetes/blob/f9354bf6b7ea0f4d57fb78d04baccde15b050d4b/test/kubemark/start-kubemark.sh#L84
However, it does not use now. 
Therefore, I think that create-certs() in pre-existing/util.sh can be removed.